returns/methods/__init__.py

```diff
@@ -1,4 +1,5 @@
 from returns.methods.cond import cond as cond
+from returns.methods.gather import gather as gather
```

Comment:
Let's create a new file: `async_.py` and put it there.

Reply:
Is naming of the method good?

Reply:
yeap